app/src/main/java/org/simple/clinic/summary/PatientSummaryViewRenderer.kt

```diff
@@ -23,27 +23,27 @@ class PatientSummaryViewRenderer(
 
   private fun setupUiForTeleconsult(model: PatientSummaryModel) {
     if (model.isTeleconsultationEnabled && model.isUserLoggedIn) {
-      ui.showContactDoctorButton()
-
-      when (model.teleconsultInfo) {
-        is TeleconsultInfo.Fetched -> {
-          ui.showContactDoctorButtonTextAndIcon()
-          ui.enableContactDoctorButton()
-        }
-        is TeleconsultInfo.MissingPhoneNumber, is TeleconsultInfo.NetworkError -> {
-          ui.showContactDoctorButtonTextAndIcon()
-          ui.disableContactDoctorButton()
-        }
-        is TeleconsultInfo.Fetching -> {
-          ui.showContactButtonProgress()
-          ui.enableContactDoctorButton()
-        }
-      }
+      renderContactDoctorButton(model)
     } else {
       ui.hideContactDoctorButton()
     }
   }
 
+  private fun renderContactDoctorButton(model: PatientSummaryModel) {
+    ui.showContactDoctorButton()
+    when (model.teleconsultInfo) {
+      is TeleconsultInfo.Fetched -> {
+        ui.contactDoctorButtonEnabled()
+      }
+      is TeleconsultInfo.MissingPhoneNumber, is TeleconsultInfo.NetworkError -> {
+        ui.contactDoctorButtonDisabled()
+      }
+      is TeleconsultInfo.Fetching -> {
+        ui.contactDoctorButtonInProgress()
+      }
```

Comment:
We can remove the braces on the `when` clause patterns now.

Reply:
Done